src/aml/bert.py

```diff
@@ -0,0 +1,203 @@
+from typing import Optional, Tuple, List, TypedDict, Dict
+import os
+import re
+import random
+from argparse import Namespace
+import pandas as pd
+from aml.mdl import AbstractAspectModel
+from bert_e2e_absa import train, work
+
+from cmn.review import Review
+
+#--------------------------------------------------------------------------------------------------
+# Typings
+#--------------------------------------------------------------------------------------------------
+class FoldItem(TypedDict):
+    train: List[int]
+    valid: List[int]
+
+class Split(TypedDict):
+    folds: Dict[str, FoldItem]
+    test: List[int]
+
+#--------------------------------------------------------------------------------------------------
+# Utilities
+#--------------------------------------------------------------------------------------------------
+def convert_reviews_from_lady(original_reviews: List[Review]) -> Tuple[List[str], List[List[str]]]:
+    reviews_list = []
+    label_list = []
+
+    # Due to model cannot handle long sentences, we need to filter out long sentences
+    REVIEW_MAX_LENGTH = 511
+
+    for r in original_reviews:
+        if not len(r.aos[0]):
+            continue
+        else:
+            aos_list = []
+
+            for aos_instance in r.aos[0]:
```

Comment:
@3ripleM pls make it one line

Reply:
@3ripleM 
I respect your opinion and it's true that **long** lines become confusing. However, we try to make it one line as far as they're not long :) pls follow our lab's convention. Thank you.

Reply:
@hosseinfani 
Whenever the characters exceed 80 chars they are long (and it is not only my opinion).